tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "references": [
     { "path": "packages/internal" },
     { "path": "packages/structure" },
-    { "path": "packages/core" },
```

Comment:
OK, maybe I'm following now: Core is now just webpack config, which means it doesn't need to be built.

Also wondering if you're thinking of deprecating or removing Core based on these changes. If so, what about the other way around — Turn Internal into Core, Turn Core into Webpack, and then we can avoid a breaking change.

fwiw...